api/tests/views_tests.py

```diff
@@ -1,12 +1,25 @@
+from datetime import datetime
 import pytest
 from model_bakery import baker
+import responses
 
 from django.contrib.auth.models import User
-from django.test import override_settings
+from django.core.cache import cache
+from django.test import (
+    override_settings,
+    RequestFactory,
+    TestCase,
+)
 from django.utils import timezone
 from django.urls import reverse
 from rest_framework.test import APIClient
 
+from api import authentication
+from api.authentication import get_cache_key
+from api.tests.authentication_tests import (
+    _setup_fxa_response,
+    _setup_fxa_response_no_json,
```

Comment:
question (non-blocking): do we (i.e., also @jwhitlock ) care about the number of places we're importing functions that start with the `_` character, which usually signals that the function is only used in the same module that defines it?

Reply:
[PEP recommends](https://peps.python.org/pep-0008/#method-names-and-instance-variables) single underscore in front of methods like:
> Use one leading underscore only for non-public methods and instance variables.

The helper functions are for non-public, for tests only, so using them like this makes sense. However, if we continue to use the helper function outside of where it is defined I think it could be better suited in a separate module that lists the helper functions for a given app. Whichever we decide on, I recommend documenting decision or further conversation in [MPP-3157 Jira ticket](https://mozilla-hub.atlassian.net/browse/MPP-3157).